obal/data/roles/diff_brew_package/tasks/dist.yml

```diff
@@ -0,0 +1,44 @@
+- name: "Set branch checks"
+  set_fact:
+    "{{ item.name }}_branch_exists": "{{ true if item.releaser in releasers else false }}"
```

Comment:
I don't know if the latter is pure python but in pure python ```item.releaser in releasers``` already returns a boolean.

Reply:
Yeah, artifact of quick copy-past-edit'in.  Latest push JUST got the python filter working